gdocs.js

```diff
@@ -39,7 +39,7 @@ if (flag === '--login') {
 }
 
 function help() {
-  console.log('Synopsys');
+  console.log('Synopsis');
```

Comment:
This is a functional change to the documentation system. If you need such changes split out, I can, but ...